client/src/components/Sidebar/Sidebar.test.js

```diff
@@ -93,7 +96,8 @@ describe('workbench sidebar', () => {
       mockStore({
         workbench: { status: { debug: true } },
         notifications,
-        ticketCreate
+        ticketCreate,
+        workbench: { config: { showApps: true } }
```

Comment:
__Question:__ There is already a `workbench` in _this_ `mockStore` object. Should there be only one (with the properties of each)? _I think two might be okay (because perhaps the object properties are merged)._

Reply:
Thank you!